src/components/ComingSoonSection/ComingSoonMovies.vue

```diff
@@ -0,0 +1,80 @@
+<script>
+import { defineComponent } from "vue";
+import MovieCardBasic from "@/components/Global/MovieCardBasic.vue";
+
+export default defineComponent({
+  name: "ComingSoonMovies",
+  data() {
+    return {
+      movies: [
+        {
+          id: 1,
+          title: "Harry Potter",
+          genre: {
+            id: 3,
+            name: "Fantasy",
+          },
+          poster_url:
+            "https://images-na.ssl-images-amazon.com/images/I/713KEd-8jyL._AC_SL1500_.jpg",
+          length: 183,
+          release_date: "2001-11-16",
+          description:
+            "An orphaned boy enrolls in a school of wizardry, where he learns the truth about himself, his family and the terrible evil that haunts the magical world.",
+        },
+        {
+          id: 2,
+          title: "Harry Potter and the Philosopher's Stone",
+          genre: {
+            id: 3,
+            name: "Fantasy",
+          },
+          poster_url:
+            "https://images-na.ssl-images-amazon.com/images/I/713KEd-8jyL._AC_SL1500_.jpg",
+          length: 159,
+          release_date: "2001-11-16",
+          description:
+            "An orphaned boy enrolls in a school of wizardry, where he learns the truth about himself, his family and the terrible evil that haunts the magical world.",
+        },
+        {
+          id: 3,
+          title: "Harry Potter and the Philosopher's",
+          genre: {
+            id: 3,
+            name: "Fantasy",
+          },
+          poster_url:
+            "https://images-na.ssl-images-amazon.com/images/I/713KEd-8jyL._AC_SL1500_.jpg",
+          length: 159,
+          release_date: "2001-11-16",
+          description:
+            "An orphaned boy enrolls in a school of wizardry, where he learns the truth about himself, his family and the terrible evil that haunts the magical world.",
+        },
+      ],
+    };
+  },
+
+  components: { MovieCardBasic },
```

Comment:
This is a very tiny thing, but a good practice nonetheless - to keep the different options we pass (in the options API) in certain order (for example `components` should be first, `data` second). You could even configure your linter to enforce that project-wide.
I won't say this is necessary (because it's opinionted) but here you can read more about this practice and what's the suggested order:
https://vuejs.org/style-guide/rules-recommended.html#component-instance-options-order